attunement.lic

```diff
@@ -24,9 +25,13 @@ class Attunement
       return
     end
 
-    room_list = if @stationary_skills_only || DRStats.necromancer?
+    room_list = if @stationary_skills_only || (DRStats.necromancer? and !@force_power_walk)
```

Comment:
Could this be better handled by using a waggle_set of ```eotb?``` instead of a new setting?

If necromancer and waggle_set !exist do